### ISSUE TITLE
Strange error: --clip returns error when running in tmux

### ISSUE THREAD

**Issue by duckpuppy** [justwatchcom/gopass#318]
I have a strange error that used to work until recently.  I'm running under macOS Sierra, tmux 2.5, and gopass 1.3.2.  

I've been using commands like `gopass --clip work/ad` in my workflow for a while now, and haven't had any issues until recently.  I always run under tmux as well.  In the past couple of days, I've seen a new error only under tmux.  The above command (or any that use `--clip`) just outputs `Error: exit status 1`.  `gopass work/ad` (without `--clip`) works fine and displays the password.  Outside of tmux everything works as expected even with the `--clip` option.  

Standard `pass` also has a similar issue... it says it has copied the password to the clipboard when running under tmux, but it hasn't.  Again, it works outside of tmux.  Apparently something has changed about accessing the system clipboard from tmux.  According to homebrew, tmux hasn't changed on my system since 6/14/2017, so it's not a tmux update.

I have no real idea of where to go about troubleshooting this, and pass/gopass are the only things having this issue that I have found.

**Comment by dominikschulz** [comment]
We're using `github.com/atotto/clipboard` to access the clipboard. This lib uses `pbcopy` on MacOS.
Can you check if using pbcopy manually works from within / outside of tmux?

**Comment by duckpuppy** [comment]
I think we can close this, but I'm still not sure what the issue was.  I did try pbcopy inside and outside of tmux, and it worked fine.  I also use the tmux plugin manager for installing tmux plugins, and that's where my problem was.  Somehow, all my plugins (other than tpm) had been deleted.  I reinstalled the plugins using tpm and restarted tmux and everything works.  I'm still not sure where the problem arose, since pbcopy itself worked just fine inside tmux while `github.com/atotto/clipboard` didn't (I was testing with the `gocopy` command provided with that package).

So, it was an issue with my tmux config, but I'm still confused as to why it was a problem.  I'll close this issue, though, since things are working now.
